unit/testing-utils/load_java_class.cpp

```diff
@@ -43,6 +45,8 @@ symbol_tablet load_java_class(
   // Configure the path loading
   cmdlinet command_line;
   command_line.set("java-cp-include-files", class_path);
+  if(use_string_refinement)
```

Comment:
This doesn't actually work - `cmdlinet` will ignore all calls to `set` if the command line parameter hasn't been "pre-registered" with it. Good news, I also recently hit this problem and if you rebase to resolve your merge conflict, you will have a new class called `free_form_cmdlinet` which allows turning on arbitrary flags and then passing the full bundle in (introduced in #1913 and used in #1918)